examples/SampleApp/android/app/proguard-rules.pro

```diff
@@ -1,3 +1,5 @@
+# Fixes the issue with React native reanimated release build
+-keep class com.swmansion.reanimated.** { *; }
```

Comment:
This is better kept on line 71.. to have rules for reanimated in the same section